app/src/main/java/org/simple/clinic/bp/BloodPressureRepository.kt

```diff
@@ -127,4 +127,8 @@ class BloodPressureRepository @Inject constructor(
       dao.save(listOf(deletedBloodPressureMeasurement))
     }
   }
+
+  fun deletedMeasurementAsStream(bloodPressureMeasurementUuid: UUID): Observable<BloodPressureMeasurement> {
+    return dao.getOneWhichIsDeleted(bloodPressureMeasurementUuid).toObservable()
```

Comment:
Wow, `getOneWhichIsDeleted()` sounds like English. What do you think about just `getOneDeleted()`? :p